src/main/java/games/strategy/triplea/TripleA.java

```diff
@@ -134,7 +133,7 @@ public void startGame(final IGame game, final Set<IGamePlayer> players, final bo
           frame.toFront();
         });
       } catch (final InterruptedException e) {
-        ClientLogger.logQuietly(e);
+        Thread.currentThread().interrupt();
```

Comment:
We have a `SwingAction` library that takes care of the `InterruptedException` in this case, right? I'm noting that as it seems like this should be handled by the the invoked code. If all we do is ignore the exception and interrupt the current thread, that would be a good detail to tuck into a library call, and would remove a `try/catch` here.

Reply:
Good call.  97f08c396